cli/src/main.rs

```diff
@@ -152,12 +56,12 @@ pub fn main() {
         Err(e) => {
             eprintln!("Error during startup: {}", e);
             error!("Error during startup: {}", e);
-            match e.raw_os_error() {
-                Some(13) => {
-                    startup_helpers::report_permission_error();
-                },
-                _ => {}
-            }
+            if cfg!(target_os = "unix" ) {
+                match e.raw_os_error() {
+                    Some(13) => report_permission_error(),
+                    _        => eprintln!("Something else!")
```

Comment:
I'd removed this. Printing this without context causes problems. Please delete this